### PR TITLE
Improve Izhikevich tutorial

### LINKED ISSUE #602: Clean up Izhikevich tutorial
- [ ] Remove installations instructions; link to docs instead
- [ ] Remove Slides.pptx and Slides.pdf
- [ ] Move content from last two slides into izhikevich_tutorial.rst:
    - [ ]  NESTML finish the Izhikevich neuron model
    - [ ] PyNESTML exercise running the neuron for different sets of parameters

### PR DESCRIPTION
**Pull request by clinssen**
Fixes #602.

### CODE REVIEW

**Comment by clinssen** [comment]
The diff is a bit messy because it is based on #618; that PR should be merged before this one.

The only file that needs to be reviewed in this PR is ``doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb``.

Cheers!

File: doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    "install_nest(\"/tmp/nestml-component\", NEST_SIMULATOR_INSTALL_LOCATION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the generated log output for any potential error messages or warnings.\n",
+    "\n",
+    "The generated module is called \"nestmlmodule\" by default. It can be loaded using nest.Install():"
```
Comment (pnbabu) [inline doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb:117]: nestmlmodule and nest.Install() could be enclosed in backquotes.

**Review by pnbabu — COMMENTED** [review]
Some minor comments.

File: doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb [hunk 2]
```diff
@@ -193,13 +185,32 @@
...
+   "metadata": {},
+   "source": [
+    "Task 2: Parameter space exploration\n",
+    "-----------------------------------\n",
+    "\n",
+    "Perform a parameter space exploration to reproduce the bottom eight panels from [1], figure 2."
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (pnbabu) [inline doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb:199]: Is this code block really required?

**Review by pnbabu — COMMENTED** [review]
Some minor comments.
Could you also remove the extra `_` on [this line](https://github.com/nest/nestml/blob/6e540aecabecc65965c5e3e4d59bdf2b02b14ae1/doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb#L52)?

File: doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb (hunk 2 above)
Comment (clinssen) [reply doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb:199]: Not strictly speaking, but it helps to show where you are expected to write some code. If you feel it's better without, I will remove it.

Comment (pnbabu) [reply doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb:199]: It looks somewhat like an incomplete code block to me, even though that is the motive of having this code block. Maybe you could add a comment something like: `# Write your code here`?

Comment (clinssen) [reply doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb]: Code block has been removed :)

Comment (pnbabu) [reply doc/tutorials/izhikevich/nestml_izhikevich_tutorial.ipynb]: Thanks, sorry to be nitpicky about this!

**Review by pnbabu — APPROVED** [review]
Looks good to me!

**Comment by clinssen** [comment]
Thanks for the feedback!

**Pull request merged**